mongo/datadog_checks/mongo/mongo.py

```diff
@@ -25,6 +25,21 @@
 ALLOWED_CUSTOM_QUERIES_COMMANDS = ['aggregate', 'count', 'find']
 
 
+def build_url(scheme, host, path='/', username=None, password=None, query_params=None):
```

Comment:
(nit) I think it's better to move that in a separate `utils.py` file. That will keep `mongo.py` cleaner and easier to read.

Reply:
Yes, agreed